src/css/home.css

```
@@ -2,807 +2,2275 @@ body.article.home {
-  width: 8px;
+html[data-theme="dark"] .home-intent-card-icon {
+  background: rgba(99, 102, 241, 0.15);
+  color: var(--indigo-400-new, #818cf8);
 }
 
-.conversation-area::-webkit-scrollbar-track {
-  background: transparent;
+/* Brand variant for ADP - uses color from page-header-data via --adp-color */
+.home-intent-card-icon--brand,
+.home-intent-card-icon.home-intent-card-icon--brand {
+  background: color-mix(in sRGB, var(--adp-color) 8%, transparent);
```

Stylelint (17.11.0)

[error] Expected "sRGB" to be "srgb" (value-keyword-case)

(value-keyword-case)